src/pages/Home.vue

```diff
@@ -21,9 +21,10 @@
         :purposeParams="getPurposeParams"
         :groupData="groupData"
         :group="getGroup"
-        :authType="authType"
+        :authType="getAuthType"
         :sessionId="sessionId"
         :userIpAddress="getUserIpAddress"
+        :extraInputValidation="isExtraInputValidationsRequired"
```

Comment:
can you also keep the prop's name to `isExtraInputValidationsRequired`?
because both variables are actually boolean, so it makes it easier for someone reading the code to understand that it is a boolean variable

Reply:
Done.